code/game/turfs/turf.dm

```diff
@@ -52,6 +52,16 @@
 	CalculateAdjacentTurfs()
 
 /turf/Destroy(force)
+	. = QDEL_HINT_IWILLGC
+	if(force)
+		..()
+		//this will completely wipe turf state
+		var/turf/basic/B = new /turf/basic(src)
```

Comment:
This is actually a premptive refactor for deleting a zlevel. Nothing actually uses it at the moment